<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>GxE Browser</title>

    <!-- Bootstrap -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/gxebrowser.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>
  <body>

    <!-- Add a navbar. Use jQuery to insert navbar from 'navigation.html' -->
    <nav class="navbar navbar-default navbar-fixed-top" id="nav-placeholder">
    </nav>

    <!-- Breadcrumbs -->
    <div class="breadcrumb-container">
      <ol class="breadcrumb" aria-label="breadcrumbs">
        <li role="menuitem"><a href="index.html">Home</a></li>
        <li role="menuitem" class="active">Tables</li>
      </ol>
    </div>

    <!-- Page Content -->
    <div class="container">

      <h1>Supplemental Tables</h1>      

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S1.xlsx">Table S1: Treatments used for analysis.</a></h4>
        <p class="caption">The control shown for each treatment is the vehicle used to dilute it. For example,
        dexamethasone was a powder resuspended in EtOH, so we used EtOH as control for the dexamethasone treatment. Note that we also matched the concentration of the vehicle used. In the case of all the treatments with EtOH as control, both the treatment and the control wells received 1ul of EtOH per 10,000&micro;L of culturing media. ”In range” denotes whether the treatment concentration matches physiological levels in the blood (within 10x), as indicated by Mayo Clinic or the Center for Disease Control. When this information was not available, we used literature reports and we indicate the relevant PMID.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S2.xlsx">Table S2: Summary of shallow sequencing.</a></h4>
        <p class="caption">Plate ID, Cell Type, Cell Line, Barcode ID (see Section 4), Treatment ID (see Table S1), Total reads (total number of reads sequenced), Aligned reads (number of reads after initial alignment), Quality reads (number of reads passing quality filtering), Clean reads (number of reads after duplicate removal).</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S3.xlsx">Table S3: Number of differentially expressed genes identified by shallow sequencing.</a></h4>
        <p class="caption">For each treatment, shown is the number of differentially expressed genes for each cell type at 10% BH-FDR, with an &#124;logFC&#124; &gt; 0.25. Differentially expressed genes are determined by contrasting a treatment with its matched control.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S4.xlsx">Table S4: Summary of deep sequencing.</a></h4>
        <p class="caption">Total reads, total number of reads sequenced. Aligned reads, number of reads after initial alignment. Quality reads, number of reads passing quality filtering. Clean reads, number of reads after allele-specific read mapping and duplicate removal via WASP.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S5.xlsx">Table S5: Number of differentially expressed genes identified by deep sequencing.</a></h4>
        <p class="caption">For each treatment, shown is the number of genes tested (”All”), the number of differentially expressed genes in each cell type at 10% BH-FDR, with an &#124;logFC&#124; &gt; 0.25, and the number of differentially expressed genes (DEG) both up- and downregulated relative to the control. DEG are determined by contrasting a treatment with its matched control. Included is also the number of DEG when contrasting Control 1 versus Control 2 in each plate separately.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S6.tar.gz">Table S6: Differentially expressed genes across treatments and cell lines.</a></h4>
        <p class="caption">For each cell type/treatment combination, given is the Ensembl transcript ID, DESeq2 p-value for differential expression, BH-adjusted p-value for differential expression, log2 fold-change relative to the control, Ensembl gene ID, and Ensembl gene name.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S7.xlsx">Table S7: Module assignments for network genes.</a></h4>
        <p class="caption">For each module, listed is the t-score for the treatments in the cell type indicated. Also included for each module is the number of genes, the assigned treatment (based on the highest average t-score across cell types), the number of heterozygous and ASE SNPs in genes within the module, and the proportion of ASE to heterozygous SNPs in genes within the module.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S8.xlsx">Table S8: Z-scores for module-treatment combinations.</a></h4>
        <p class="caption"></p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S9">Table S9: Summary of ASE in deep sequenced samples.</a></h4>
        <p class="caption">For each treatment (row) and individual (columns), listed is the number of ASE SNPs at 10% FDR, with the total number of heterozygous SNPs tested in parentheses.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S10.txt.gz">Table S10: ASE results for all SNPs tested.</a></h4>
        <p class="caption">All heterozygous SNPs covered by at least 40 reads are listed, along with the cell type and treatment condition. Also listed are the number of reads covering the reference and alternate alleles, the ASE &beta;, standard error, p-value from QuASAR, and the q-value.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S11.xlsx">Table S11: SNPs displaying conditional ASE.</a></h4>
        <p class="caption">For each cASE SNP, listed is the SNP rsID, cell-type, gene, treatment, the &Delta;AST Z-score, empirical p-value, and q-value, the MeSH control-only BF, empirical p-value, and q-value, and the MeSH treatment-only BF, empirical p-value, and q-value. Significant cASE is defined as having q-value < 0.24 (24% FDR) from at least one of the tests.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S12.txt.gz">Table S12: cASE results for all SNPs tested.</a></h4>
        <p class="caption">For each SNP tested for cASE, listed is the same information in S11, with two additional columns: the SNP-based expression and SNP-based log( fold change ), described in Section 9.3.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S13.xlsx">Table S13: cASE genes with a reQTL signal.</a></h4>
        <p class="caption">Out of 83 genes reported to have GxE in previous reQTL and ASE studies and able to be tested for cASE in our data, 63 genes are listed in this table that also have cASE in our data (p-value <0.05). The table denotes the genes, the environment in which the reQTL was identified, the citation of the paper establishing the reQTL, and the cell-types and treatments for which we find cASE.</p>
      </div>

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S14.xlsx">Table S14: Heterogeneity values for each cell-type/treatment subgroup.</a></h4>
        <p class="caption">Heterogeneity and confidence intervals are calculated as described in Section 10.4. Also shown for each subgroup is the number of significant ASE and cASE SNPs.</p>
      </div>      

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S15.xlsx">Table S15: SNPs displaying induced ASE.</a></h4>
        <p class="caption">Each iASE SNP is listed with the inducing treatment and matched control, cell type, and gene. Also given is the adjusted read coverages (see Section 10.5) for the reference and alternate allele in both treatment and control conditions.</p>
      </div>      

      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S16.xlsx">Table S16: Summary of GWAS meta-analysis traits examined.</a></h4>
        <p class="caption">Shown for each trait is the trait abbreviation and the citation for the meta analysis study.</p>
      </div>      
      
      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S17.xlsx">Table S17: GEMMA per SNP heritability estimates relative to the genomic average.</a></h4>
        <p class="caption">Shown for each GWAS trait tested (see Table S16) are the estimates for cASE/iASE (SNPs in genic regions with cASE or iASE), ASE (SNPs in genic regions with ASE), No ASE (SNPs in genic regions without ASE) and Intergenic (SNPs farther than 100kb from any gene). PVE: proportion of variance in phenotypes explained.</p>
      </div>      
      
      <div class="well">
        <h4 class="caption"><a href="Tables/Supplemental_Table_S18.xlsx">Table S18: Overlap of GWAS-identified genes and genes containing cASE or iASE SNPs.</a></h4>
        <p class="caption">The table shows genes that have previously been associated with a phenotype through GWAS and contain evidence of cASE (77 genes with 85 SNPs) or iASE (28 genes with 35 SNPs). Also denoted are the treatment and cell-type for which we identified cASE.</p>
      </div>

    </div> <!-- /container -->

    <!-- Include necessary javaScript files/libraries -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/gxebrowser.js"></script>
    <script src="js/analytics.js"></script>
  </body>
</html>
